packages/blade/src/components/Alert/Alert.tsx

```diff
@@ -111,7 +117,7 @@ const intentIconMap = {
   notice: AlertIcon,
 };
 
-const Alert = ({
+export const Alert = ({
```

Comment:
Let's export at the end, easier to track what we are exporting.

Reply:
Export the AlertProps as well.

Reply:
Done in https://github.com/razorpay/blade/pull/666/commits/f2ae55867de2cc17e9f039fa0cd99e49df089d25